10/umbraco-commerce/release-notes.md

```diff
@@ -17,6 +17,13 @@ If you are upgrading to a new major version, check the breaking changes in the [
 
 This section contains the release notes for Umbraco Commerce 10 including all changes for this version.
 
+#### [10.0.12](https://github.com/umbraco/Umbraco.Commerce.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F10.0.12) (July 3rd 2024)
+
+* Added pesimistic locking to payment provider callback endpoint to prevent concurrency issues if the endpoint is called too multiple times at once [#533](https://github.com/umbraco/Umbraco.Commerce.Issues/issues/533).
+* Fixed percentage discounts not taking the stores rounding method into account during calculation [#506](https://github.com/umbraco/Umbraco.Commerce.Issues/issues/506).
+* Fixed issue where order lines with a zero value would cause a concurrency exception due to the fact their prices aren't frozen but the order recalculation process was attempting to refreeze them.
+* Updated Order properties to trim whitespace around values to prevent unexpected behaviour  [#528](https://github.com/umbraco/Umbraco.Commerce.Issues/issues/528).
```

Comment:
```suggestion
* Updated Order properties to trim whitespace around values to prevent unexpected behavior [#528](https://github.com/umbraco/Umbraco.Commerce.Issues/issues/528).
```